test/test_ssl.rb

```diff
@@ -1222,6 +1222,49 @@ def test_get_ephemeral_key
     end
   end
 
+  def test_fallback_scsv
+    ctx_proc = proc { |ctx|
+      ctx.ciphers = "kRSA"
```

Comment:
Avoid kRSA since it will not work with TLS 1.3-capable OpenSSL. It's not finalized yet, but I try to keep the tests pass with the master branch of OpenSSL.

Reply:
Fixed